PWGHF/D2H/TableProducer/candidateCreatorB0Reduced.cxx

```diff
@@ -115,6 +135,122 @@ struct HfCandidateCreatorB0Reduced {
     setLabelHistoCands(hCandidates);
   }
 
+  /// Main function to perform B0 candidate creation
+  /// \param withDmesMl is the flag to use the table with ML scores for the D- daughter (only possible if present in the derived data)
+  /// \param collision the collision
+  /// \param candsDThisColl B0 candidates in this collision
+  /// \param tracksPionThisCollision pion tracks in this collision
+  /// \param invMass2DPiMin minimum B0 invariant-mass
+  /// \param invMass2DPiMax maximum B0 invariant-mass
+  template <bool withDmesMl, typename Cands, typename Pions, typename Coll>
+  void runCandidateCreationDStar(Coll const& collision,
+                                 Cands const& candsDThisColl,
+                                 Pions const& tracksPionThisCollision,
+                                 const float& invMass2DPiMin,
+                                 const float& invMass2DPiMax)
```

Comment:
I don't see `invMass2DPiMin` nor `invMass2DPiMax` modified inside the function. Pass them by value.